packages/cli/src/zosfiles/compare/doc/ICompareFileOptions.ts

```diff
@@ -0,0 +1,24 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * These are the options to be used in file comaparison to handle the
+ *  binary, encoding, record and volumeSerial operations in the comparison process
+ *
+ * @exports
+ * @interface ICompareFileOptions
+ */
+export interface ICompareFileOptions {
+    binary?: boolean | false;
+    encoding?: string | "";
+    record?: boolean | false;
+    volumeSerial?: string | "";
+}
```

Comment:
Can the `| false` and `| ""` be removed from these type definitions? I'm not sure what their intent is, since interface properties can't have default values.

To set default values for these properties, you could do it in the constructor of the class, and optionally include comments in this file that mention the default values as well 🙂